### ISSUE TITLE
RU conda package recipe - fix dependencies

### ISSUE THREAD

**Issue by mtitov** [radical-cybertools/radical.utils#321]
List of dependencies in conda-forge should be updated ([current list](https://github.com/conda-forge/radical.utils-feedstock/blob/master/recipe/meta.yaml#L26-36))
Should be
```yaml
    - python
    - colorama
    - msgpack-python
    - netifaces
    - ntplib
    - pymongo < 4
    - pyzmq
    - regex
    - setproctitle
```

Package `radical.gtod` is not in conda-forge, should it be added there and as dependency for RU?

**Comment by andre-merzky** [comment]
> Package `radical.gtod` is not in conda-forge, should it be added there and as dependency for RU?

Yes, that sounds sensible.

**Comment by mtitov** [comment]
fixed in PR [#64](https://github.com/conda-forge/radical.utils-feedstock/pull/64) - ticket will be closed after merging changes
